.eslintrc.json

```diff
@@ -4,21 +4,24 @@
     "es2021": true,
     "jest": true
   },
-  "parser": "babel-eslint",
+  "parser": "@babel/eslint-parser",
   "extends": [
```

Comment:
I can't start your project because of `eslint` errors, I think you should add such changes in `eslintrc.json`
![Untitled](https://user-images.githubusercontent.com/52933810/155120187-673a679b-6307-46a0-9e27-22fa69cecb43.png)